classify.py

```diff
@@ -106,6 +110,9 @@ def description(self, classes):
 
         return (first_guess+" "+second_guess).strip()
 
+    def frames(self):
```

Comment:
`num_frames` would be better? I would expect something called `frames` to actually return frame data

Reply:
You could also make this a read-only property like this:

```python
@property
def num_frames(self):
    return len(self.prediction_history)
```

Which is accessed like this:

```python
prediction.num_frames
```

Reply:
Ha ha.   I agree with all these new names - I was going alone with the style of the current code!